core/src/main/java/org/apache/carbondata/core/util/ValueCompressionUtil.java

```diff
@@ -78,26 +78,26 @@ private ValueCompressionUtil() {
   private static DataType getDataType(double value, int decimal, byte dataTypeSelected) {
     DataType dataType = DataType.DATA_DOUBLE;
     if (decimal == 0) {
-      if (value < Byte.MAX_VALUE) {
+      if (value < Byte.MAX_VALUE && value > Byte.MIN_VALUE) {
```

Comment:
Before i modified to use new "absMaxValue", it is needed, but now i can detele this code.